### ISSUE TITLE
update README and debugging questions 

### ISSUE THREAD

**Issue by melisandeteng** [blutjens/geospatial_unet_pytorch#1]
- update instruction in readme  to `git clone  https://github.com/blutjens/earth_obs_seg.git`
- I don't understand this line: https://github.com/blutjens/earth_obs_seg/blob/f553ba51bd10faaedffae0a2b6343f424512505a/earth_obs_seg/eval/online_eval.py#L49 I think the multiplication by the batch size is not necessary? 

- Unless I missed something there is no loss.backward() call or optimizer.step() call here ? https://github.com/blutjens/earth_obs_seg/blob/main/earth_obs_seg/train.py

**Comment by blutjens** [comment]
- I actually need to update this to instruct people to fork the repo rather than clone it. 
- The 'loss' variable contains the average loss per image in the batch. To compute the total_loss we multiple the loss with the number of images in the batch. 
- wow! such a good catch. I had accidentally deleted backward() because the original code base used a gradient scalar and I had deleted it.

**Comment by blutjens** [comment]
fixed.
